web/src/components/contributorlist.jsx

```diff
@@ -3,25 +3,29 @@ import { useParams } from 'react-router-dom';
 
 import zonesConfig from '../../../config/zones.json';
 
-const getContributorId = (contributor) => {
-  const contributorSplitted = contributor.split('/');
-  return contributorSplitted[contributorSplitted.length-1];
-}
-
 const ContributorList = () => {
   const { zoneId } = useParams();
-  const contributors = (zonesConfig[zoneId] || {}).contributors || [];
 
   return (
     <div className="contributors">
-      {contributors.map(contributor => {
-        const contributorId = getContributorId(contributor);
-    
+      {zonesConfig[zoneId]?.contributors?.map((contributor) => {
         return (
-          <a key={contributorId} href={contributor} rel="noopener noreferrer" target="_blank">
-            <img src={`https://avatars.githubusercontent.com/${contributorId}?s=40`} alt={contributorId} />
+          <a
+            key={contributor}
+            href={`https://github.com/${contributor}`}
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <img
+              src={`https://avatars.githubusercontent.com/${contributor}?s=20`} // loads the avatar image at a default size of 20px
+              srcSet={`https://avatars.githubusercontent.com/${contributor}?s=40 2x`} // loads the avatar image at a default size of 40px for high resolution displays
+              alt={contributor}
+              height="20px"
+              width="20px"
```

Comment:
Are you sure using `px` in these attributes is allowed?

Reply:
Oh crap you are right, according to MDN they are supposed to be without units...
It works just fine with for me with units in Firefox and Chrome though but might be best to change it.